packages/cms-data-sync/src/calendars/sync-all.ts

```diff
@@ -0,0 +1,46 @@
+import { Logger } from '@asap-hub/server-common';
+import { CalendarDataProvider } from '@asap-hub/model';
+
+import { fetchContentfulEntries, getContentfulClient } from '../utils';
+import { rateLimiter } from '../utils/rate-limiter';
+
+export async function syncCalendars({
+  dataProvider,
+  logger,
+}: {
+  dataProvider: CalendarDataProvider;
+  logger: Logger;
+}) {
+  const { contentfulEnvironment } = await getContentfulClient();
+
+  const calendars = await fetchContentfulEntries(
```

Comment:
can't you use the data provider here too?

Reply:
This function ensures we get all the calendar, i don't think it makes sense to duplicate the function just so it uses the data provider.